fbgemm_gpu/codegen/embedding_backward_code_generator.py

```diff
@@ -11,6 +11,7 @@
 from typing import Any, Dict, List, Optional, Tuple, Union
 
 import jinja2
+import torch
```

Comment:
Sorry, another thing. Can we make this script PyTorch independent (i.e., remove `import torch`)? We do not always run this code generator after fully installing PyTorch. It'd be nice if this script can take an argument from CMake when it runs.

Reply:
Perhaps here you can pass another ROCm-specific argument:
https://github.com/pytorch/FBGEMM/blob/5ce897d4bd46a59aac5f93459239c206f20ef15f/fbgemm_gpu/CMakeLists.txt#L174-L175

Reply:
Or an internal comment from @ajtulloch This should be easier to implement and maintain.

> Can’t you just do 4 * kWarpSize (or C10_WARP_SIZE or w/e the compile time warp size constant is) and avoid having to put this logic in the template system?